src/Entity/InvoiceShopBillingData.php

```diff
@@ -45,64 +44,64 @@ public function getCompany(): ?string
         return $this->company;
     }
 
-    public function getTaxId(): ?string
+    public function setCompany(?string $company): void
```

Comment:
Methods re-arranged in order of class variables declaration/doctrine entities declaration